source/core/security-encryption-at-rest.txt

```diff
@@ -99,6 +99,25 @@ transport encryption.
 
 For details, see :ref:`rotate-encryption-keys`.
 
+Logging
+~~~~~~~
+
+.. versionadded:: 3.4 Available in MongoDB Enterprise only
```

Comment:
Just wondering:  Given that under the Encrypted Storage Engine header we state that the feature is Enterprise only, do we need to reiterate the enterprise availability here as well?